ci/jenkins/test.sh

```diff
@@ -1172,6 +1187,21 @@ EOF
     done
 }
 
+function configure_kind_ipam_routes {
+    
+   control_plane_podcidr=$(kubectl get nodes $KIND_CLUSTER-control-plane -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+   control_plane_node_ip=$(kubectl get nodes $KIND_CLUSTER-control-plane  -o wide | awk 'FNR == 2 {print$6}')
+   sudo ip route add $control_plane_podcidr via $control_plane_node_ip
+  
+   worker_podcidr=$(kubectl get nodes $KIND_CLUSTER-worker -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+   worker_node_ip=$(kubectl get nodes $KIND_CLUSTER-worker  -o wide | awk 'FNR == 2 {print$6}')
+   sudo ip route add $worker_podcidr via $worker_node_ip
+
+   worker2_podcidr=$(kubectl get nodes $KIND_CLUSTER-worker2 -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+   worker2_node_ip=$(kubectl get nodes $KIND_CLUSTER-worker2  -o wide | awk 'FNR == 2 {print$6}')
+   sudo ip route add $worker2_podcidr via $worker2_node_ip
+}
+
```

Comment:
can make this function generic wrt cluster and node type
ex
local cluster="$1"
    local node_type="$2"
    
    local pod_cidr=$(kubectl get nodes "${cluster}-${node_type}" -o yaml | grep podCIDR | awk 'FNR == 1 {print $2}')
    local node_ip=$(kubectl get nodes "${cluster}-${node_type}" -o wide | awk 'FNR == 2 {print $6}')
    
    sudo ip route add "$pod_cidr" via "$node_ip"